### PR TITLE
[RU] Update Support Team

### PR DESCRIPTION
**Pull request by TicClick**

### CODE REVIEW

File: wiki/People/The_Team/Support_Team/ru.md [hunk 1]
```diff
@@ -1,37 +1,25 @@
----
-outdated: true
----
+# Команда поддержки
 
-# Техподдержка
+**Команда поддержки** (иногда **Support Team Redux**) — группа пользователей, модерирующих следующие форумы: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (за исключением [Tournaments](https://osu.ppy.sh/community/forums/55) и [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4) и [Help](https://osu.ppy.sh/community/forums/5).
 
-**Техподдержка** (Support Team Redux, «Техподдержка Возрождённая» — [Ephemeral](https://osu.ppy.sh/users/Ephemeral), 2014) — группа, участники которой следят за следующими форумами:
+Команду поддержки **не стоит путать** со службой поддержки (или просто техподдержкой), которая решает вопросы, связанные с аккаунтами и рестриктами, и отвечает по адресам [support@ppy.sh](mailto:support@ppy.sh) и [accounts@ppy.sh](mailto:support@ppy.sh).
```
Comment (cl8n) [inline wiki/People/The_Team/Support_Team/ru.md:5]: this paragraph isn't present in English, is it something only appropriate for a translation?

File: wiki/People/The_Team/Support_Team/ru.md [hunk 2]
```diff
@@ -1,37 +1,25 @@
...
-- [Skinning](https://osu.ppy.sh/community/forums/15) ([скининг](/wiki/Skinning));
-- [Feature Requests](https://osu.ppy.sh/community/forums/4) (нововведения);
-- [Help](https://osu.ppy.sh/community/forums/5) (техподдержка).
+## Права и обязанности
 
-## Роли и обязанности
+Команда поддержки отвечает за две вещи:
 
-**Внимание: эти люди НЕ отвечают на вопросы о сайленсах, нарушении правил и тому подобных вещах.** Для этого пишите на accounts@ppy.sh.
+1. Проверка и подтверждение сообщений об ошибках ([баг-репортов](https://ru.wikipedia.org/wiki/Программная_ошибка)), которые оставляют пользователи.
```
Comment (cl8n) [inline wiki/People/The_Team/Support_Team/ru.md:11]: same with this wikipedia link

File: wiki/People/The_Team/Support_Team/ru.md [hunk 3]
```diff
@@ -1,37 +1,25 @@
----
-outdated: true
----
+# Команда поддержки
 
-# Техподдержка
+**Команда поддержки** (иногда **Support Team Redux**) — группа пользователей, модерирующих следующие форумы: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (за исключением [Tournaments](https://osu.ppy.sh/community/forums/55) и [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4) и [Help](https://osu.ppy.sh/community/forums/5).
```
Comment (UristMcKobold) [inline wiki/People/The_Team/Support_Team/ru.md:3]:
```suggestion
**Команда поддержки** (иногда **Support Team Redux**) — группа пользователей, модерирующих следующие подфорумы: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (за исключением [Tournaments](https://osu.ppy.sh/community/forums/55) и [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4) и [Help](https://osu.ppy.sh/community/forums/5).
```

File: wiki/People/The_Team/Support_Team/ru.md (hunk 1 above)
Comment (TicClick) [reply wiki/People/The_Team/Support_Team/ru.md:5]: not really, just the wording that's more clear and straightforward. I'll make the same change to the original soon

File: wiki/People/The_Team/Support_Team/ru.md [hunk 4]
```diff
@@ -1,37 +1,25 @@
...
-- [Feature Requests](https://osu.ppy.sh/community/forums/4) (нововведения);
-- [Help](https://osu.ppy.sh/community/forums/5) (техподдержка).
+## Права и обязанности
 
-## Роли и обязанности
+Команда поддержки отвечает за две вещи:
 
-**Внимание: эти люди НЕ отвечают на вопросы о сайленсах, нарушении правил и тому подобных вещах.** Для этого пишите на accounts@ppy.sh.
+1. Проверка и подтверждение сообщений об ошибках ([баг-репортов](https://ru.wikipedia.org/wiki/Программная_ошибка)), которые оставляют пользователи.
+2. Организация тем в форумах [Feature Requests](https://osu.ppy.sh/community/forums/4) и [Help](https://osu.ppy.sh/community/forums/5), которые в итоге помечаются одним из следующих тегов: *Resolved* (решённая проблема), *Confirmed* (существующая проблема), *Invalid* (некорректная жалоба), *Duplicate* (жалоба-дубликат) и/или *Added* (реализованная просьба).
```
Comment (UristMcKobold) [inline wiki/People/The_Team/Support_Team/ru.md:12]:
```suggestion
2. Организация тем в подфорумах [Feature Requests](https://osu.ppy.sh/community/forums/4) и [Help](https://osu.ppy.sh/community/forums/5), которые в итоге помечаются одним из следующих тегов: *Resolved* (решённая проблема), *Confirmed* (подтверждённая проблема), *Invalid* (некорректная жалоба), *Duplicate* (жалоба-дубликат) и/или *Added* (реализованная просьба).
```
«Существующая проблема» звучит так, будто это дубликат.

File: wiki/People/The_Team/Support_Team/ru.md [hunk 5]
```diff
@@ -1,37 +1,25 @@
...
 
-Техподдержка отвечает за:
+Если вы нуждаетесь в помощи, и ваша проблема относится к теме одного из перечисленных форумов, создайте там новую тему, и с вами свяжутся. Помощь на русском языке можно получить в теме **«[Вопросы и проблемы](https://osu.ppy.sh/community/forums/topics/9668)»**.
```
Comment (UristMcKobold) [inline wiki/People/The_Team/Support_Team/ru.md:14]:
```suggestion
Если вы нуждаетесь в помощи, и ваша проблема относится к теме одного из перечисленных подфорумов, создайте там новую тему, с вами свяжутся. Помощь на русском языке можно получить в теме **«[Вопросы и проблемы](https://osu.ppy.sh/community/forums/topics/9668)»**.
```
Слишком много союзов.

**Review by UristMcKobold — CHANGES_REQUESTED** [review]

**Review by UristMcKobold — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**